Pratiksha/html/jobportal/Assignment8.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Assignment8</title>
+    <link rel="stylesheet" href="Assignment8.css" />
+  </head>
+  <body>
+    <div>
+      <div class="cls_wrapper">
+        <p>
```

Comment:
aap ne p tag ke andar kyu img tag ko dala hau

Reply:
ye tarika galat hai Sir

Reply:
pratiksha remove this and push file again